boot/zephyr/boards/nrf52840dk_nrf52840_ram_load.overlay

```diff
@@ -0,0 +1,26 @@
+/ {
+	sram@2003FC00 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x2003FC00 DT_SIZE_K(1)>;
```

Comment:
```suggestion
	sram@2003FF00 {
		compatible = "zephyr,memory-region", "mmio-sram";
		reg = <0x2003FF00 0x100>;
```

Reply:
@danieldegrasse Can you respond or make changes, so that we can go through with this PR?

Reply:
Sure, should be fixed now